src/panels/Menu/TopMenuBar/Menus/ViewMenu.tsx

```diff
@@ -13,9 +14,32 @@ import { TopBarMenuWrapper } from '../TopBarMenuWrapper';
 
 export function ViewMenu() {
   const { menuItems, setMenuItemVisible } = useMenuItems();
-
+  const [propertyVisibility, setPropertyVisibility] = useGetOptionPropertyValue(
+    'OpenSpaceEngine.PropertyVisibility'
+  );
+  const description = useGetPropertyDescription('OpenSpaceEngine.PropertyVisibility');
   const dispatch = useAppDispatch();
 
+  if (!description) {
+    return <></>;
+  }
+
+  const { Options: data } = description.additionalData as AdditionalDataOptions;
+
+  // Get the name of the option, e.g. "Option 1"
+  // Flatten the array as otherwise each element is an array
+  // @TODO (ylvse 2025-03-18): Change the property format
+  const options = data.map((option) => Object.values(option)).flat();
+
+  function onChange(option: string | null) {
+    if (option && options.indexOf(option) !== -1) {
+      // Now we need to find the number key of the option
+      // which is the same as its index in the optionsStrings array
+      const index = options.indexOf(option);
+      setPropertyVisibility(index);
+    }
+  }
```

Comment:
This is off by one, the map from OpenSpace is: 

0: Object { 1: "Novice User" }
1: Object { 2: "User" }
2: Object { 3: "Advanced User" }
3: Object { 4: "Developer" }
4: Object { 5: "Everything" }

But with the `options` we've mapped it 0-4 instead so Everything -> Developer, Developer -> Advanced User which also leads to this error when setting Novice User 
(E) OptionProperty       Could not find an option for value '0'
(W) Logging              Number of Errors: 1

Reply:
Would also put the index on line 38 outside the if so we don't do the same `option.indexOf(..)`